Botón de ayuda implementado

adriantd3 opened the pull request.

adriantd3 committed:
Cambios colores y boton help

- Estilos de TopBar modificados para establecer los colores con variables globales.
- Nuevo icono de ayuda añadido.
- Popup de ayuda con descripción añadido.

adriantd3 committed:
helpButton - Small fix

VctPerez commented on Dutify/src/components/topBar/helpButton/helpButton.jsx:
```diff
@@ -0,0 +1,22 @@
+import { GrCircleQuestion } from "react-icons/gr";
+import "../iconsStyle.css";
+
+function HelpButton({ visible }) {
```
Solo esta en modo claro. Yo le pondría modo oscuro para que haya consistencia con lo demás.

VctPerez commented on Dutify/src/components/helpModal/helpModal.jsx:
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+function HelpModal() {
```
Solo está en modo claro. Yo le pondría también los colores del modo oscuro para mantener consistencia

adriantd3 replied:
Hecho

VctPerez commented on Dutify/src/components/helpModal/helpModal.jsx:
```diff
@@ -0,0 +1,54 @@
+            <ul>
+              <li>Buscar y reproducir canciones.</li>
+              <li>Reproducir canciones en modo aleatorio o repetición.</li>
+              <li>Crear y editar nuevas listas de reproducción.</li>
```
Crear, editar y eliminar *

DavidBuenoCarmona replied:
Aunque eliminar sea una de las cosas que se puede hacer con las listas, no creo que se tenga que poner a la misma altura que crear y editar que si son acciones principales de la aplicación. Ya que un usuario usara Dutify para Crear y Editar, se puede eliminar pero no creo que sea tan principal

adriantd3 replied:
Lo he dejado por ahora en eliminar

VctPerez commented on Dutify/src/components/helpModal/helpModal.jsx:
```diff
@@ -0,0 +1,54 @@
+            </ul>
+          </div>
+          <div className="modal-footer">
+            <button
```
![image](https://github.com/JoseBueno30/Dutify/assets/124668589/32216b8c-3d73-4371-bfd9-845b1898d93d)
Veo redundante alguna de las dos opciones para cerrar la pestaña, quitaría alguna.
- Si dejas la x, ponla un poco más oscura de normal, en el modo claro, para que se diferencie mejor.

adriantd3 replied:
He dejado la x

DavidBuenoCarmona commented on Dutify/src/components/topBar/helpButton/helpButton.jsx:
```diff
@@ -0,0 +1,22 @@
+function HelpButton({ visible }) {
+  return (
+    <div
+      style={{ aspectRatio: "1/1", height: "80%" }}
```
No deberíamos tener Css directamente sobre el html. Yo lo sacaría a un archivo css en esa misma carpeta. Por mantener el orden en el proyecto y tener las cosas localizables

adriantd3 replied:
He arreglado eso

DavidBuenoCarmona commented on Dutify/src/App.css:
```diff
@@ -5,7 +5,7 @@
   --color-text:white;
   --border: none;
-  --color-theme:lightskyblue;
+  --color-theme:rgb(51, 162, 253);
```
Esto se ha cambiado también en la tarea de los modos (Claro y oscuro), te dará conflicto al mergear. El que esta en la otra tarea es el mismo color que tenemos en el figma. Comprueba si es el mismo que has puesto tu

adriantd3 replied:
Lo he puesto por el grupo

DavidBuenoCarmona commented on Dutify/src/App.css:
```diff
@@ -15,5 +15,5 @@
   --color-text:black;
   --border: none;
-  --color-theme:lightskyblue;
+  --color-theme:rgb(51, 162, 253);
```
Esto se ha cambiado también en la tarea de los modos (Claro y oscuro), te dará conflicto al mergear. El que esta en la otra tarea es el mismo color que tenemos en el figma. Comprueba si es el mismo que has puesto tu

adriantd3 replied:
Lo mismo

DavidBuenoCarmona commented on Dutify/src/components/topBar/topBarStyle.css:
```diff
@@ -2,33 +2,17 @@
+    border-color: var(--color-3);
 }
 
-#light .topBar{
```
Esto ha sido cambiado también el la otra tarea, revisalo al mergear

DavidBuenoCarmona reviewed (commented):
Mergea la main en esta rama antes de terminar el PR

adriantd3 committed:
helpButton y TopBar - Arreglos

- Arreglo texto helpModal.
- Tema claro y oscuro helpModal.
- Color theme actualizado.
- Arreglos estilos topBar.

VctPerez commented on Dutify/src/App.css:
```diff
@@ -5,7 +5,7 @@
   --color-text:white;
   --border: none;
-  --color-theme:lightskyblue;
+  --color-theme:#623CEA;
```
En modo oscuro hay elementos que son dificilmente distinguibles

VctPerez commented on Dutify/src/components/topBar/topBarStyle.css:
```diff
@@ -2,33 +2,17 @@
 
-#dark .topBar{
-    background-color: #4E4848;
+.btn-back{
```
En modo claro la flecha no debe ser blanca + el fondo tampoco se distingue muy bien

adriantd3 committed:
TopBar- Update estilo

VctPerez commented:
El botón de la flecha no he mirado por qué es un rectangulo en vez de un cuadrado, pero eso no es para solucionarlo aquí

adriantd3 committed:
Merge branch 'main' into atd-helpButton
